app/frontend/src/cljs/teet/task/task_view.cljs

```diff
@@ -479,8 +609,8 @@
     [:span]))
 
 (defn- task-page-content
-  [e! app activity {status :task/status :as task} pm? files-form]
-  [:div.task-page
+  [e! app activity {status :task/status :as task} pm? assignee? files-form]
+  [:div.task-page {:style {:margin-bottom "2em"}}
```

Comment:
did you mean to use em instead of rem? we have a class helper in common-styles/margin-bottom which could be used here